src/main/java/buchen/fallingsand/Main.java

```diff
@@ -0,0 +1,23 @@
+package buchen.fallingsand;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        try {
+            Sand sand = new Sand(Integer.parseInt(args[0]), Integer.parseInt(args[1]));
+            sand.randomSand(Integer.parseInt(args[2]));
```

Comment:
I never said to take command line arguments for these things.
I gave you specific values.